users/vieting/experiments/switchboard/ctc/feat/experiments.py

```diff
@@ -618,13 +993,137 @@ def run_mel_audio_perturbation():
+        {"tempo": {"prob": 1, "minimum": 0.7, "maximum": 1.3}},
+        {"speed": {"prob": 0.5, "minimum": 0.88, "maximum": 1.12}},
+        {"speed": {"prob": 0.3, "minimum": 0.88, "maximum": 1.12}},
+        {"speed": {"prob": 0.7, "minimum": 0.88, "maximum": 1.12}},
+        {"speed": {"prob": 0.6, "minimum": 0.8, "maximum": 1.2}},
+        {"speed": {"prob": 0.6, "minimum": 0.9, "maximum": 1.1}},
+        {"speed": {"prob": 1, "minimum": 0.9, "maximum": 1.1}},
+        {"speed": {"prob": 1, "minimum": 0.8, "maximum": 1.2}},
+        {"non_linearity": {"prob": 0.7, "minimum": 0.9, "maximum": 1.1}},
+        {"non_linearity": {"prob": 0.3, "minimum": 0.9, "maximum": 1.1}},
+        {"non_linearity": {"prob": 1, "minimum": 0.9, "maximum": 1.1}},
+        {"preemphasis": {"prob": 0.3, "minimum": 0.94, "maximum": 1.0, "default": 0.97}},
+        {"preemphasis": {"prob": 0.7, "minimum": 0.94, "maximum": 1.0, "default": 0.97}},
+        {"preemphasis": {"prob": 1.0, "minimum": 0.94, "maximum": 1.0, "default": 0.97}},
+        {"preemphasis": {"prob": 0.7, "minimum": 0.90, "maximum": 1.0, "default": 0.97}},
+        {"preemphasis": {"prob": 0.7, "minimum": 0.90, "maximum": 1.0, "default": 0.95}},
+    ]
+
+    nn_base_args = {}
+
+    for args in perturbation_args:
+        exp_name_suffix, report_args = args_to_key_and_report_strings(args)
+
+        # Construct the exp_name and report_args
+        exp_name = f"mel_bs2x5k_perturb_from_checkpoint_24_{exp_name_suffix}"
+        nn_base_args[exp_name] = dict(
+            returnn_args={
+                "extra_args": {
+                    "audio_perturb_args": args,
+                    "audio_perturb_runner": CodeWrapper("WaveformPerturbation(**audio_perturb_args)"),
+                    "conv_pad_seq_len_to_power": 1.5,
+                    "accum_grad_multiple_step": 2,
+                    "preload_from_files": {
+                        "existing-model": {
+                            "filename": nn_system.train_jobs["conformer_bs2x5k_lgm80_baseline"].out_checkpoints[24],
+                            "init_for_train": True,
+                        }
+                    },
+                },
+                **returnn_args,
+            },
+            feature_args={
+                "class": "LogMelNetwork",
+                "wave_norm": True,
+                "frame_size": 200,
+                "frame_shift": 80,
+                "fft_size": 256,
+            },
+            lr_args=lr_args,
+            report_args=report_args,
+        )
+
+    nn_args, report_args_collection = get_nn_args_baseline(
+        nn_base_args=nn_base_args,
+        num_epochs=426,
+        evaluation_epochs=[376, 390, 400, 426],
+        prefix="conformer_",
+    )
+
+    returnn_root = CloneGitRepositoryJob(
+        "https://github.com/rwth-i6/returnn",
+        commit="c4d36d06f6465e82a50d400d114259e07b8b0709",
+    ).out_repository
+    returnn_root.hash_overwrite = "returnn_conv_padding"
```

Comment:
```suggestion
```
This is not needed.

Reply:
Can I remove this for all experiments or only this one?

Reply:
Only for log Mel where you don't need the conv padding option. It doesn't matter too much though, if you keep it it doesn't harm.